static/src/stylesheets/module/crosswords/_clues.scss

```diff
@@ -42,10 +42,6 @@
 
     .has-grouped-clues & {
         width: $gs-gutter * 2;
-
-        @include mq(leftCol) {
-            width: $gs-gutter * 3;
-        }
```

Comment:
The new design calls for tighter clue boxes to fit the ad column. Removes the extra space at the leftCol breakpoint as well so it's consistent at larger sizes.